letterboxd/build.gradle.kts

```diff
@@ -0,0 +1,33 @@
+@Suppress("DSL_SCOPE_VIOLATION") // TODO: Remove once KTIJ-19369 is fixed
+plugins {
+    id("java-library")
+    alias(libs.plugins.build.config)
+    alias(libs.plugins.org.jetbrains.kotlin.jvm)
+    kotlin("kapt")
+}
+
+buildConfig {
```

Comment:
neat build config plugin for non-android modules (can be used in android modules too!)